tests/src/main.cpp

```diff
@@ -101,7 +109,8 @@ namespace UDataTests {
 
 int main() {
     TestContext context;
-
+    
+    // Unit Tests
```

Comment:
I'm not sure it's wise to try to label our tests like this. Most of our "unit" tests are really just tests; they don't make much of an effort to isolate the component being tested